web-components/src/components/inputs/TextFieldBase.tsx

```diff
@@ -0,0 +1,201 @@
+import React from 'react';
+import InputAdornment from '@mui/material/InputAdornment';
+import MuiTextField, { BaseTextFieldProps } from '@mui/material/TextField';
+import { DefaultTheme as Theme, makeStyles } from '@mui/styles';
+import clsx from 'clsx';
+
+import { DefaultStyleProps } from './FieldFormControl';
+import InputLabel from './InputLabel';
+
+// TODO: create styled component as described in issue: regen-network/regen-web/issues/955
```

Comment:
see other comment for explanation on this file

Reply:
Oh haha never mind